Set broker to support websocket

hannahoprchal opened the pull request:
Configuration to make broker container accessible via web-sockets.


### Fixes

- [EXDRHUB-1232](https://issues.redhat.com/browse/EXDRHUB-1232) 

### Checklist

- [ ] Related tests were updated
- [ ] Related documentation was updated
- [ ] OpenAPI file was updated
- [ ] Run `tox`, no tests failed

guioliveirabh commented on docker-compose.yml:
```diff
@@ -106,11 +106,16 @@ services:
       retries: 5
       timeout: 30s
+    volumes:
+      - ./config/rabbitmq/rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf
+      - ./config/rabbitmq/enabled_plugins:/etc/rabbitmq/enabled_plugins
```
```suggestion
      - ./config/rabbitmq/rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf:ro
      - ./config/rabbitmq/enabled_plugins:/etc/rabbitmq/enabled_plugins:ro
```

As these are configuration files, it's safer to set them as read-only.

guioliveirabh commented on config/rabbitmq/enabled_plugins:
```diff
@@ -0,0 +1 @@
+[rabbitmq_management, rabbitmq_web_stomp, rabbitmq_stomp].
```
```suggestion
[rabbitmq_management, rabbitmq_web_stomp, rabbitmq_stomp].

```
New line at end of file will make next diffs look better.

guioliveirabh commented on config/rabbitmq/rabbitmq.conf:
```diff
@@ -0,0 +1,3 @@
+#more https://www.rabbitmq.com/configure.html#config-file
+
+web_stomp.tcp.port = 15674
```
```suggestion
web_stomp.tcp.port = 15674

```
New line at end of file will make next diffs look better.

guioliveirabh commented on tox.ini:
```diff
@@ -70,7 +70,7 @@ ignore =
     tests/**
     tox.ini
-
+    config/rabbitmq/*
```
Actually, you can't ignore these configuration files. You need to include them on the rhub-api package.
You can do that by including them under: [MANIFEST.in](https://github.com/resource-hub-dev/rhub-api/blob/master/MANIFEST.in)

fpob replied:
These configs are only for rabbitmq container, not for rhub-api.

guioliveirabh reviewed (commented):
Almost everything looks good to me, but I suggest for you to add more people, as I'm mostly dedicated to Beaker, I may be missing something.

hannahoprchal commented:
Hi @guioliveirabh,
thank you for your suggestions I addressed them into the code and squashed it with other commits.
Feel free to re-review this once you have time.

Thanks,
Hana
